src/MutationInterface.php

```diff
@@ -33,4 +33,6 @@ public function getOriginalFileAst(): array;
     public function isOnFunctionSignature(): bool;
 
     public function isCoveredByTest(): bool;
+
+    public function getMutatedNode();
```

Comment:
How about adding a return type here?

Reply:
added, but had to revert. See how NodeTraverser works: https://github.com/nikic/PHP-Parser/blob/7f96481c80ecfe0fd8d37506064d3a25ed9cad23/lib/PhpParser/NodeTraverser.php#L209-L229

We can have many return types, including `Node`, `Node[]`, `int`, `null`.

I've added PHPDoc with all of them instead.